pkg/services/ngalert/api/api.go

```diff
@@ -38,6 +44,7 @@ type API struct {
 	Schedule        schedule.ScheduleService
 	Store           store.Store
 	DataProxy       *datasourceproxy.DatasourceProxyService
+	Alertmanager    Alertmanager
```

Comment:
Is it required? I don't think it's used.

Reply:
It is, otherwise the interface the interface is unused.

Reply:
what do you mean unused?

Reply:
The interface we defined above which is mean to serve the API struct - I mean, I can remove it but then we have no way to signal where this interface is meant to be used which can be a source of confusion.

Reply:
We will use it soon in future PRs, I guess?

Reply:
I was expecting @papagian to use in this alertmanager API implementation PR.

Reply:
ok, I see what you mean now! I was thinking of calling this in store package when storing the configuration but maybe I can move it in the API level...